Documentation/configuring-hive-metastore.md

```diff
@@ -1,11 +1,38 @@
 # Configuring the Hive metastore
 
-Generally the default configuration of Hive metastore works for small clusters, but users may wish to improve performance or move storage requirements out of cluster by using a dedicated database for storing table metadata for Presto and Hive server.
+Hive metastore is responsible for storing all the metadata about the database tables we create in Presto and Hive.
+By default, the metastore stores this information in a local embedded Derby database in a PersistentVolume attached to the pod.
 
-## Use MySQL or Postgresql for the Hive Metastore database
+Generally the default configuration of Hive metastore works for small clusters, but users may wish to improve performance or move storage requirements out of cluster by using a dedicated SQL database for storing the Hive metastore data.
+
+## Configuring PersistentVolumes
+
+Hive by default requires 1 Persistent Volume to operate.
+
+`hive-metastore-db-data` is the main PVC required by default.
+It is used by Hive metastore to retain information about the location of where data is stored, which Presto and Hive server use.
+In practice, it is possible to remove this requirement by using [MySQL](#use-mysql-for-the-hive-metastore-database) or [PostgreSQL](#use-postgresql-for-the-hive-metastore-database) for the Hive metastore database.
+
+To install Hive metastore requires that dynamic volume provisioning be enabled via a Storage Class, a persistent volumes of the correct size must be manually pre-created, or that you use a pre-existing MySQL or PostgreSQL database.
```

Comment:
"To install, Hive metastore ..."

Reply:
Isn't it actually "To install Hive metastore, requires..."?

Reply:
Actually, I could see as yours, and as it is currently, my previous suggestion seems wrong though when I say it outloud.